### PR TITLE
Tile Endpoint Change + Small UI fixes (tooltip + topbar region name)

### PR DESCRIPTION
**Pull request by sachiyuan**
- Made Tile Endpoint more generalized and add region to call when it is made
- Small tooltip edit
- Boundary end point fix
- Edit html of top bar so "Southern California" doesn't get cut off by drop down arrow

### CODE REVIEW

File: src/interface/src/app/services/map.service.ts [hunk 1]
```diff
@@ -104,6 +105,8 @@ export class MapService {
         return 'north_coast_inland';
       case Region.SOUTHERN_CALIFORNIA:
         return 'southern_california';
+      case null:
+        return '';
```
Comment (runforyourlives) [inline src/interface/src/app/services/map.service.ts:109]: Should this default to Sierra Nevada (see two lines below)?

File: src/interface/src/app/map/map.component.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
       <!-- Upload a planning area -->
       <div class="upload-wrapper">
-        <div matTooltip="This item is not available in this version." [matTooltipDisabled]="login_enabled">
+        <div matTooltip="This feature is not available at this time." [matTooltipDisabled]="login_enabled">
```
Comment (runforyourlives) [inline src/interface/src/app/map/map.component.html:44]: This is a different change; ok to leave in, but please note the change in the PR description and title.

Comment (sachiyuan) [reply src/interface/src/app/map/map.component.html:44]: Good point — fixed the title!

File: src/interface/src/app/services/map.service.ts (hunk 1 above)
Comment (sachiyuan) [reply src/interface/src/app/services/map.service.ts]: Ooh thank you for catching this — fixed

**Pull request merged**